examples/virtual_keyboard/ui/icons.slint

```diff
@@ -0,0 +1,14 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+// Forkawesome Icons:
+// Copyright © 2018 Dave Gandy & Fork Awesome
+// License MIT
+export global Icons {
+    out property <string> arrow-up: "M1611 971c0 34-14 66-37 90l-75 75c-24 24-57 38-91 38s-67-14-90-38l-294-293v704c0 72-60 117-128 117H768c-68 0-128-45-128-117V843l-294 293c-23 24-56 38-90 38s-67-14-90-38l-75-75c-24-24-38-56-38-90s14-67 38-91l651-651c23-24 56-37 90-37s67 13 91 37l651 651c23 24 37 57 37 91z";
```

Comment:
should that just be svg images maybe?  Then we would also have support in no_std.